apps/.storybook/config.js

```diff
@@ -12,6 +13,22 @@ import '../style/netsim/style.scss';
 import '../style/applab/style.scss';
 import '../src/templates/GameButtons.story.scss';
 
+Props.propTypes = {
```

Comment:
Looks like this is described as a temporary fix in the github issue - I think a comment explaining this add would be helpful.